### ISSUE TITLE
Topic:Feature Enhancements

### ISSUE THREAD

**Issue by Harshita9412** [MinavKaria/Ratna-Supermarket#32]
### Describe the types of topics you'd add in our Website

Hi @MinavKaria ,

I’d like to propose a few enhancements that could improve the functionality and user experience of the project.
1) Auto Fetch Location:
This would enhance usability by streamlining the user experience when accessing location-based services.
2) Backend Functionality for Login and Signup

Currently, the products within the same categories are not well organized. Enhancing the organization of these products into their respective sections would improve navigation and user satisfaction.
-> Kids Section and Pets Food Section:
     This could attract more users looking for family-friendly options.
->Kitchen and Grocery Section:
   This would add significant value to users, enabling them to find everything they need in one place.
   Home Page Transition Video:

3) Adding a transition video showcasing food or swapping images on the home page could create a more engaging and visually appealing user experience.

I believe these enhancements could greatly benefit the user experience and overall functionality of the project. 

 I would love to take on the implementation of these features.

### Record

- [x] I agree to follow this project's Code of Conduct
- [x] I'm a GSSOC-24-extd contributor
- [x] Soon, I'll complete it and create a Pull Request.

**Comment by MinavKaria** [comment]
Hi @Harshita9412! Thanks for opening this issue on Ratna Supermarket. We appreciate your contribution to this open-source project. We aim to respond or assign your issue as soon as possible.

**Comment by Harshita9412** [comment]
Thanks @MinavKaria for your response. I'm looking forward to your feedback

**Comment by MinavKaria** [comment]
I want you to only add auto fetch location right now , is that ok ?

**Comment by Harshita9412** [comment]
Yes, I’ll focus on adding the auto-fetch location feature for now.

**Comment by Harshita9412** [comment]
> I want you to only add auto fetch location right now , is that ok ?

**Comment by MinavKaria** [comment]
Hello @Harshita9412! Your issue #32 has been closed. Thank you for your contribution!🚀

**Comment by techy4shri** [comment]
@Harshita9412 you need to reopen this issue.
The follow the following steps:

1. Fork the repo, copy only the main branch.
2. Clone the repo on local machine, i.e. your computer with `git clone https://github.com/Harshita9412/Ratna-Supermarket.`
3. Create a branch `git checkout -b <new branch name>`.
4. work on the solution, create fileor whatever.
5.  Commit your changes: `git add.` then `git commit -m "enter commit message"`
6. `git push`
7. Now compare and pull. Make the PR according to the template.

@MinavKaria I think Harshita misunderstood, she is new here, kindly reassign her and she can work on it.

**Comment by MinavKaria** [comment]
Hello @Harshita9412! Your issue #32 has been closed. Thank you for your contribution!🚀

**Comment by Harshita9412** [comment]
> @Harshita9412 you need to reopen this issue. The follow the following steps:
> 
> 1. Fork the repo, copy only the main branch.
> 2. Clone the repo on local machine, i.e. your computer with `git clone https://github.com/Harshita9412/Ratna-Supermarket.`
> 3. Create a branch `git checkout -b <new branch name>`.
> 4. work on the solution, create fileor whatever.
> 5. Commit your changes: `git add.` then `git commit -m "enter commit message"`
> 6. `git push`
> 7. Now compare and pull. Make the PR according to the template.
> 
> @MinavKaria I think Harshita misunderstood, she is new here, kindly reassign her and she can work on it.

@techy4shri Thankyou for the guidance!

**Comment by techy4shri** [comment]
Happy to help.

**Comment by Harshita9412** [comment]
> I want you to only add auto fetch location right now , is that ok ?

@MinavKaria please assign this to me.

**Comment by Harshita9412** [comment]
> I want you to only add auto fetch location right now , is that ok ?

@MinavKaria please re assign this task to me.

**Comment by techy4shri** [comment]
@Harshita9412 you might have better luck getting noticed by raising a new issue, referencing this one and messaging the PA on discord.
